### DISCUSSION TITLE
Update to Newer Versions

### DISCUSSION THREAD

**Discussion by dreamercast** [LibreHardwareMonitor/LibreHardwareMonitor#1003] [Q&A]
Hi guys, just wondering if there's an easy way to upgrade to the latest builds (nightly or otherwise)?  Is there a way to save my widget setup in case of version upgrade or a need to re-install?  Thanks!

**Comment by thenotoriousberg** [comment]
Hey there,

Just figured this out myself as I too didn't want to reconfigure everything.

In the installation folder where the older version of librehardwaremonitor.exe is, copy librehardwaremonitor.CONFIG, then paste this file into the newly downloaded and unzipped Libre Hardware Monitor folder. The program will automatically locate this when started so that's all that is required. 

After, you can delete the older version's install folder to keep things tidy.
